pkg/validate/container.go

```diff
@@ -251,3 +303,55 @@ func testExecSync(c internalapi.RuntimeService, containerID string) {
 	Expect(stderr).To(BeNil(), "The stderr should be nil.")
 	framework.Logf("Execsync succeed")
 }
+
+// createLogContainer creates a container with log and the prefix of containerName.
+func createLogContainer(c internalapi.RuntimeService, prefix string, podID string, podConfig *runtimeapi.PodSandboxConfig) (string, string) {
+	By("create a container with log and name")
+	containerName := prefix + framework.NewUUID()
+	path := fmt.Sprintf("%s.log", containerName)
+	containerConfig := &runtimeapi.ContainerConfig{
+		Metadata: buildContainerMetadata(containerName, defaultAttempt),
+		Image:    &runtimeapi.ImageSpec{Image: defaultContainerImage},
+		Command:  []string{"echo", defaultLog},
+		LogPath:  path,
+	}
+	containerID, err := c.CreateContainer(podID, containerConfig, podConfig)
+	framework.ExpectNoError(err, "failed to create container: %v", err)
+	framework.Logf("Created container %s\n", containerID)
+	return containerConfig.LogPath, containerID
+}
+
+// parseDockerJSONLog parses logs in Docker JSON log format. Docker JSON log format
+// example:
+//   {"log":"content 1","stream":"stdout","time":"2016-10-20T18:39:20.57606443Z"}
+//   {"log":"content 2","stream":"stderr","time":"2016-10-20T18:39:20.57606444Z"}
+func parseDockerJSONLog(log []byte, msg *logMessage) {
```

Comment:
I have updated the PR. I think the log in CRI format is STRING type, but I'm not sure. So please review it. @Random-Liu